src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -703,3 +702,57 @@ def get_streaming_metrics_consumer(
         Topic(topic),
         processing_factory,
     )
+
+
+class LastSeenUpdaterMessageFilter(StreamMessageFilter[KafkaPayload]):  # type: ignore
+    def should_drop(self, message: KafkaPayload) -> bool:
+        return False
+
+
+class LastSeenUpdaterCollector(ProcessingStrategy[int]):  # type: ignore
```

Comment:
I would put all this consumer in its own module. It does not have to live with the indexer consumer.